telegram/bot.py

```diff
@@ -1125,32 +1214,41 @@ def answer_callback_query(self,
                               cache_time=None,
                               timeout=None,
                               **kwargs):
-        """Use this method to send answers to callback queries sent from inline keyboards. The
-        answer will be displayed to the user as a notification at the top of the chat screen or as
-        an alert.
+        """
+        Use this method to send answers to callback queries sent from inline keyboards. The answer
+        will be displayed to the user as a notification at the top of the chat screen or as an
+        alert.
+        Alternatively, the user can be redirected to the specified Game URL. For this option to
+        work, you must first create a game for your bot via BotFather and accept the terms.
+        Otherwise, you may use links like t.me/your_bot?start=XXXX that open your bot with
+        a parameter.
 
         Args:
-            callback_query_id (str): Unique identifier for the query to be answered.
-            text (Optional[str]): Text of the notification. If not specified, nothing will be shown
-                to the user.
-            show_alert (Optional[bool]): If `True`, an alert will be shown by the client instead of
-                a notification at the top of the chat screen. Defaults to `False`.
-            url (Optional[str]): URL that will be opened by the user's client.
-            cache_time (Optional[int]): The maximum amount of time in seconds that the result of
-                the callback query may be cached client-side. Telegram apps will support caching
-                starting in version 3.14. Defaults to 0.
-            timeout (Optional[int|float]): If this value is specified, use it as the read timeout
-                from the server (instead of the one specified during creation of the connection
-                pool).
-            **kwargs (dict): Arbitrary keyword arguments.
+            callback_query_id (:obj:`str`): Unique identifier for the query to be answered.
+            text (:obj:`str`, optional): Text of the notification. If not specified, nothing will
+                be shown to the user, 0-200 characters.
+            show_alert (:obj:`bool`, optional): If true, an alert will be shown by the client
+                instead of a notification at the top of the chat screen. Defaults to false.
+            url (:obj:`str`, optional): URL that will be opened by the user's client. If you have
+                created a Game and accepted the conditions via @Botfather, specify the URL that
+                opens your game - note that this will only work if the query comes from a callback
+                game button. Otherwise, you may use links like t.me/your_bot?start=XXXX that open
+                your bot with a parameter.
+            cache_time (:obj:`int`, optional): The maximum amount of time in seconds that the
+                result of the callback query may be cached client-side. Telegram apps will support
```

Comment:
it's in tg's docs and so it is in ours. https://core.telegram.org/bots/api#answercallbackquery